packages/mobile/src/fiatExchanges/ExternalExchanges.tsx

```diff
@@ -0,0 +1,111 @@
+import ListItem from '@celo/react-components/components/ListItem'
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts.v2'
+import variables from '@celo/react-components/styles/variables'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { SafeAreaView, ScrollView, StyleSheet, Text, View } from 'react-native'
+import { useSelector } from 'react-redux'
+import { FiatExchangeEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import AccountNumber from 'src/components/AccountNumber'
+import BackButton from 'src/components/BackButton.v2'
+import { EXCHANGE_PROVIDER_LINKS } from 'src/config'
+import i18n from 'src/i18n'
+import LinkArrow from 'src/icons/LinkArrow'
+import { emptyHeader } from 'src/navigator/Headers.v2'
+import { navigateToURI } from 'src/utils/linking'
+import { currentAccountSelector } from 'src/web3/selectors'
+
+export const externalExchangesScreenOptions = () => {
+  return {
+    ...emptyHeader,
+    headerLeft: () => <BackButton />,
+    headerTitle: i18n.t('fiatExchangeFlow:exchanges'),
+  }
+}
+
+function ExternalExchanges() {
+  const account = useSelector(currentAccountSelector)
+
+  const goToProvider = (link: string) => {
+    return () => {
+      ValoraAnalytics.track(FiatExchangeEvents.external_exchange_link, {
+        link,
+      })
+      navigateToURI(link)
+    }
+  }
+
+  const { t } = useTranslation('fiatExchangeFlow')
+  const providers = EXCHANGE_PROVIDER_LINKS // TODO Dynamically fetch exchange provider links so they can be updated between releases
+
+  return (
+    <ScrollView style={styles.container}>
+      <SafeAreaView>
+        <Text style={styles.pleaseSelectProvider}>{t('youCanTransfer')}</Text>
+        <View style={styles.accountNumberContainer}>
+          <View style={styles.accountNoTextContainer}>
+            <Text style={styles.accountNoText}>Account</Text>
+            <Text style={styles.accountNoText}>No.</Text>
```

Comment:
I think we need to run this text through the t function to get them properly translated. I believe we also have an `AccountNumber` component purpose built for this.

Reply:
This matches what we're using in the drawer navigator, as we've chosen not to translate it: https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/navigator/DrawerNavigator.tsx#L166
The AccountNumber component doesn't contain the "Account No." text as the formatting we'd like relative to the number changes